app/models/katello/purpose_status.rb

```diff
@@ -0,0 +1,56 @@
+module Katello
+  class PurposeStatus < HostStatus::Status
+    VALID = 0
+    INVALID = 1
+    UNKNOWN = 2
+
+    SUBSTATUSES = [
+      Katello::PurposeSlaStatus,
+      Katello::PurposeRoleStatus,
+      Katello::PurposeUsageStatus,
+      Katello::PurposeAddonsStatus
+    ].freeze
+
+    def self.status_name
+      N_('System Purpose')
+    end
+
+    def self.humanized_name
+      'purpose'
+    end
+
+    def to_label(_options = {})
+      case status
+      when VALID
+        N_('Matched')
+      when INVALID
+        N_('Mismatched')
```

Comment:
Would it be possible to build a list of what's mismatched here? I'm wishing the Foreman status showed more info?

![image](https://user-images.githubusercontent.com/429763/46688242-d90adc00-cbca-11e8-9a6d-709054b4778e.png)

Reply:
Yeah that is definitely vague. The problem is that to_label is also used (in my other PR) to show this same text on the content host details where the wireframes give a one-word summary. On that page I think it'd be strange to have a longer explanation since the individual statuses are listed right below the overall status.

I'll experiment with something though!

Reply:
I think that's fine then, a user really needs to go to the content host page to see any details about subscription-related things anyway.